src/main/java/frc/trigon/robot/subsystems/pitcher/Pitcher.java

```diff
@@ -78,6 +78,14 @@ public boolean atTargetPitch() {
         return Math.abs(masterMotor.getSignal(TalonFXSignal.POSITION) - targetPitch.getRotations()) < PitcherConstants.PITCH_TOLERANCE.getRotations();
     }
 
+    void pitchToAmp() {
+        setTargetPitch(PitcherConstants.AMP_PITCH);
+    }
+
+    void setPitchForCloseShot() {
+        setTargetPitch(PitcherConstants.CLOSE_SHOT_PITCH);
+    }
```

Comment:
Honestly, instead of having these two, I would just call `getSetTargetPitchCommand` with a constant pitch.